.github/workflows/bump-version-and-create-release-and-push-docker-images.yaml

```diff
@@ -18,23 +18,13 @@ jobs:
       - name: Check out the repository
         uses: actions/checkout@v4
 
-      - name: Read version from file
-        run: |
-          # Read the version from the version file, only store the number (without the 'v')
-          INITIAL_VERSION=$(source version && echo ${VERSION#v})
-          echo "Current version: $INITIAL_VERSION"
-          echo "INITIAL_VERSION=${INITIAL_VERSION}" >> $GITHUB_ENV
-
-      - name: Bump version
+      - name: Compute new version
         id: bump_version
-        uses: anothrNick/github-tag-action@v1
-        env:
-          DEFAULT_BUMP: minor
-          DRY_RUN: true
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          INITIAL_VERSION: ${{ env.INITIAL_VERSION }}
-          WITH_V: true
-
+        run: |
+          # Run the compute_version script and pass the PR description as an argument
+          cd .github/workflows
+          new_tag=$(./compute_version.sh "${{ github.event.pull_request.body }}")
```

Comment:
Isn't this vulnerable to code injections?

Reply:
I had originally fixed it in [fa95771](https://github.com/KIT-MRT/arbitration_graphs/pull/85/commits/fa957711c76412b5d66c09564187cb057cbd849d) but it turned out that an injection would already break things before the script is called. Therefore I already needed to sanitize the input inside the workflow, see [20e87b8](https://github.com/KIT-MRT/arbitration_graphs/pull/85/commits/20e87b82f896e43ec20d48a7d03bcaa7084420d9). It's now sanitized twice but I guess that doesn't really hurt.